plugins/user-settings-backend/src/alpha.ts

```diff
@@ -26,7 +26,7 @@ import { createRouter } from './service/router';
  * @alpha
  */
 export default createBackendPlugin({
-  pluginId: 'userSettings',
+  pluginId: 'user-settings',
```

Comment:
user setting frontend plugin is using `user-settings`.

Reply:
I think that this would be a breaking change if there's modules written on top of the new backend system, is that correct @Rugvip or maybe it doesn't matter?

Reply:
Yep it would but this plugin doesn't have any extension points so it's quite unlikely. This plugin is also just plain broken atm as well given that the frontend plugin expects the route path to be `/api/user-settings/...`